### ISSUE TITLE
Add cascadeOnCustomValue property to enable cascading ComboBox on custom value entered in the parent

### ISSUE THREAD

**Issue by IvanDanchev** [telerik/kendo-ui-core#6476]
### Reproduction of the problem
Dojo example: [https://dojo.telerik.com/ivERIreJ](https://dojo.telerik.com/ivERIreJ)
1. Focus the Categories ComboBox and type in custom text, e.g., "aaa"
2. Blur the ComboBox

### Current behavior
The "change" event of the parent fires, but the cascading ComboBox remains disabled. 

### Expected/desired behavior
The cascading ComboBox is enabled.

### Environment

* **Kendo UI version:** 2021.2.616
* **jQuery version:** x.y
* **Browser:** [all ]

**Comment by yanisslav** [comment]
Priority increased - Ticket ID 1521442

**Comment by keithmycek** [comment]
Subscribed.... and anxious for updates.

**Comment by kmycek** [comment]
Has this been resolved in a subsequent release from 2021.2.616? I tried the Dojo example, and it seems to work?

**Comment by kmycek** [comment]
Oh, no. Nevermind. It still doesn't work. Any idea when this will be fixed?

**Comment by kmycek** [comment]
BTW - I updated the Dojo script in my testing to point to 2022.2.510 to test if you might have fixed it and not updated this bug report. Sadly, it is still an outstanding bug. I could REALLY use this functioning ASAP. I implemented a convoluted and imperfect work around since I logged the bug, but now we have a need for three levels of cascading - and without the child combobox being enabled, our users cannot enter a new value for the parent(s). We have been a paid Telerik customer for many, many years now (10?). This seems like it wouldn't be an unreasonable ask for this bug to finally be fixed in a timely matter. It has been over a year since I logged it...
